2025-03-05 23:27:47,813 - prediction_model - INFO - Logging to file: C:\Users\clubs\Desktop\Prediction Model\data\Logs\prediction_model_20250305_232747.log
2025-03-05 23:27:50,606 - prediction_model - ERROR - Error calculating indicators: Cannot set a DataFrame with multiple columns to the single column Volume_Ratio
2025-03-05 23:27:50,609 - prediction_model - ERROR - Error in create_interactive_price_chart: ['date']
Traceback (most recent call last):
  File "C:\Users\clubs\Desktop\Prediction Model\src\dashboard\dashboard\dashboard_error.py", line 52, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\clubs\Desktop\Prediction Model\src\dashboard\dashboard\dashboard_visualization.py", line 96, in create_interactive_price_chart
    df = df.dropna(subset=[date_col])
  File "C:\Users\clubs\anaconda3\envs\ml_env\lib\site-packages\pandas\util\_decorators.py", line 331, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\clubs\anaconda3\envs\ml_env\lib\site-packages\pandas\core\frame.py", line 6561, in dropna
    raise KeyError(np.array(subset)[check].tolist())
KeyError: ['date']
2025-03-05 23:27:50,687 - prediction_model - ERROR - Error calculating indicators: Cannot set a DataFrame with multiple columns to the single column Volume_Ratio
2025-03-05 23:28:44,382 - prediction_model - ERROR - Error calculating indicators: Cannot set a DataFrame with multiple columns to the single column Volume_Ratio
2025-03-05 23:28:44,388 - prediction_model - ERROR - Error in create_interactive_price_chart: ['date']
Traceback (most recent call last):
  File "C:\Users\clubs\Desktop\Prediction Model\src\dashboard\dashboard\dashboard_error.py", line 52, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\clubs\Desktop\Prediction Model\src\dashboard\dashboard\dashboard_visualization.py", line 96, in create_interactive_price_chart
    df = df.dropna(subset=[date_col])
  File "C:\Users\clubs\anaconda3\envs\ml_env\lib\site-packages\pandas\util\_decorators.py", line 331, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\clubs\anaconda3\envs\ml_env\lib\site-packages\pandas\core\frame.py", line 6561, in dropna
    raise KeyError(np.array(subset)[check].tolist())
KeyError: ['date']
2025-03-05 23:28:44,482 - prediction_model - ERROR - Error calculating indicators: Cannot set a DataFrame with multiple columns to the single column Volume_Ratio
